packages/drivers/web-utils/src/utils.ts

```diff
@@ -214,6 +216,36 @@ export function removeMarkedElementsHighlights() {
   container?.remove();
 }
 
+export function waitForDomStable(
+  stabilityDelay: number = DEFAULT_POLL_INTERVAL,
```

Comment:
`stabilityDelay` → `stabilityThreshold` (clearer that this is a threshold for considering DOM stable)